parlai/scripts/convo_render.py

```diff
@@ -4,17 +4,20 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-import os
 import sys
 import json
 import random
 import argparse
 import tempfile
+import subprocess
 
 # Constants
 END_OF_CONVO = "EOC"
 CHROME_PATH = r'/Applications/Google\ Chrome.app/Contents/MacOS//Google\ Chrome'
 
+ALT_EMOJI_IMG = "https://pbs.twimg.com/media/DUzY3TpWkAAOi34.png"
```

Comment:
the user should be able to specify the icons as arguments. These are fine as defaults though. Maybe the alt could be the [facebook parrot emoji](https://emojipedia.org/facebook/4.0/parrot/) by default though :D

Reply:
Sure I can add that, but they need to be URL's because local image files don't work well when converting from html to pdf/png.

Reply:
Really? Even `file:///path/to/filename.png`?

Reply:
Hadn't tried the file approach. Thanks for that suggestion! Works with that!